README.md

```diff
@@ -31,26 +31,26 @@ installed.
 
 For each BSP version/architecture, you also need a helper script (and possibly a
 patch to apply to NVIDIA's scripts), available from the [meta-tegra repository](https://github.com/madisongh/meta-tegra).
-For L4T R32.2.3, checkout the **zeus** branch of that repository. The helper scripts
-are at `recipes-bsp/tegra-binaries/files/tegraXXX-flash-helper.sh`.  For L4T R32.3.1,
-checkout the **zeus-l4t-r32.3.1** branch, and the helper scripts are at
+For L4T R32.4.3, checkout the **dunfell** branch of that repository. The helper scripts
```

Comment:
Dunfell is at R32.3.1

Reply:
That's correct, I pushed new commit to correct this one.